.github/workflows/ci.yml

```diff
@@ -1,7 +1,6 @@
 on:
-  push:
-    branches: [ staging, trying, master ]
   pull_request:
+  merge_group:
```

Comment:
this is wrong, should keep the push.branches = master and pull_request

we want to run ci on pull requests and on merge queue